scripts/docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+version: '3'
+
+services:
+  mssql:
+    image: microsoft/mssql-server-linux:2017-latest
+    ports:
+      - '21433:1433'
+    environment:
+      - ACCEPT_EULA=Y
+      - SA_PASSWORD=S0meVeryHardPassword
+    healthcheck:
+      test: /opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P 'S0meVeryHardPassword' -Q 'select 1'
+  initmssqlknexdb:
+    image: microsoft/mssql-server-linux:2017-latest
+    links:
+      - mssql
+    depends_on:
+      - mssql
+    entrypoint:
+      - bash
+      - -c
+      - 'until /opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P S0meVeryHardPassword -d master -Q "CREATE DATABASE knex_test"; do sleep 5; done'
+
+  mysql:
+    image: mysql
+    command: --default-authentication-plugin=mysql_native_password
+    ports:
+      - '23306:3306'
+    environment:
+      - MYSQL_ROOT_PASSWORD=testrootpassword
+      - MYSQL_DATABASE=knex_test
+      - MYSQL_USER=testuser
+      - MYSQL_PASSWORD=testpassword
+    healthcheck:
+      test:
+        [
+          'CMD',
+          '/usr/bin/mysql',
+          '-hlocalhost',
+          '-utestuser',
+          '-ptestpassword',
+          '-e',
+          'SELECT 1',
+        ]
+      interval: 30s
+      timeout: 5s
+      retries: 3
+    restart: always
+  waitmysql:
+    image: mysql
+    links:
+      - mysql
+    depends_on:
+      - mysql
+    entrypoint:
+      - bash
+      - -c
+      - 'until /usr/bin/mysql -hmysql -utestuser -ptestpassword -e "SELECT 1"; do sleep 5; done'
+
+  postgres:
+    image: postgres:alpine
+    ports:
+      - '25432:5432'
+    environment:
+      - POSTGRES_USER=testuser
+      - POSTGRES_PASSWORD=knextest
+      - POSTGRES_DB=knex_test
+  waitpostgres:
+    image: postgres:10-alpine
```

Comment:
why do we need two different postgres images?

Reply:
oops